visidata/features/menu_guide.py

```diff
@@ -0,0 +1,29 @@
+from visidata import GuideSheet, vd
+
+class MenuGuide(GuideSheet):
+    guide_text = '''# The Menu System
+
+Located at the [:black on 68]very top of the screen[/], it contains a collection of commands organised within submenu trees.
```

Comment:
We should use the semantic colors (would be `[:menu]` in this case).  But I don't know about using colors inline like this anyway.  Let's drop this color.

Also, "the menu organizes many commands within submenu trees."

- Use American spellings for consistency (sorry).
- "contains" is passive voice; "contains" "collection" "organized" are all saying the same basic thing.